docs/tutorials/t2ramsey_tutorial.ipynb

```diff
@@ -0,0 +1,373 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# $T_2$ Ramsey Experiment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This experiment serves as one of the series of experiments used to characterize a single qubit. Its purpose is to determine two of the qubit's properties: *detuning frequency* and *$T_2*$*. The rough frequency of the qubit was already determined previously. Here, we would like to measure the *detuning*, that is  the difference between the qubit's precise frequency and the frequency of the rotation pulses (based on the rough frequency). This part of the experiment is also called a *Ramsey Experiment*. $T_2*$ represents the rate of decay toward a mixed state, when the qubits is initiated to the |+⟩ state."
```

Comment:
This cell is not being rendered correctly - starting at the first *$T_2*$*

Reply:
I don't see a problem here when I open it. @yaelbh , can you try?

Reply:
I didn't check on our machine, but here in GitHub it looks like this:
![image](https://user-images.githubusercontent.com/22316088/123808021-0a86e100-d8f9-11eb-8731-cecc672c8ceb.png)